rtlive/sources/data_it.py

```diff
@@ -1,39 +1,210 @@
+
+    content = requests.get(
+        "https://raw.githubusercontent.com/pcm-dpc/COVID-19/master/dati-regioni/dpc-covid19-ita-regioni-" + str(today) + ".csv",
+    ).content
+    today_data = pandas.read_csv(
+        io.StringIO(content.decode("utf-8")),
+        sep=",",
+        dtype={"codice_regione": str},
+        parse_dates=["data"],
+        usecols=["codice_regione", "data", "nuovi_positivi", "tamponi"],
+    ).rename(
+        columns={
+            "codice_regione": "region",
+            "data": "date",
+            "nuovi_positivi": "new_cases",
+            "tamponi": "new_tests",
+        }
+    )
+    today_data.set_index(["region", "date"]).sort_index()
+
+    yesterday_obj = today_obj - timedelta(days=1)
+    yesterday = yesterday_obj.strftime('%Y%m%d')
+    content = requests.get(
+        "https://raw.githubusercontent.com/pcm-dpc/COVID-19/master/dati-regioni/dpc-covid19-ita-regioni-" + str(yesterday) + ".csv",
+    ).content
+    yesterday_data = pandas.read_csv(
+        io.StringIO(content.decode("utf-8")),
+        sep=",",
+        dtype={"codice_regione": str},
+        parse_dates=["data"],
+        usecols=["codice_regione", "data", "nuovi_positivi", "tamponi"],
+    ).rename(
+        columns={
+            "codice_regione": "region",
+            "data": "date",
+            "nuovi_positivi": "new_cases",
+            "tamponi": "new_tests",
+        }
+    )
+    yesterday_data.set_index(["region", "date"]).sort_index()
+
+    for row in yesterday_data.itertuples():
+        current = today_data[today_data['region'].isin([row.region])]
+        current_tests = current['new_tests'] - row.new_tests
+        today_data.loc[current.index, 'new_tests'] = current_tests
+    
+    return today_data
+    
 
 def forecast_IT(df: pandas.DataFrame):
-    """ Applies testcount interpolation/extrapolation.
+    """ Applies testcount interpolation/extrapolation to french data.
 
     Currently this assumes the OWID data, which only has an "all" region.
     In the future, this should be replaced with more fine graned data loading!
     """
     # forecast with existing data
     df['predicted_new_tests'], results = preprocessing.predict_testcounts_all_regions(df, 'IT')
-    # interpolate the initial testing ramp-up to account for missing data
-    df_region = df.xs('all')
-    df_region.loc['2020-01-01', 'predicted_new_tests'] = 0
-    df_region.predicted_new_tests = df_region.predicted_new_tests.interpolate('linear')
-    df_region['region'] = 'all'
-    df = df_region.reset_index().set_index(['region', 'date'])    
     return df, results
 
 
 from .. import data
 data.set_country_support(
     country_alpha2="IT",
     compute_zone=data.Zone.Europe,
-    region_name={
-        "all": "Italy",
-    },
-    region_population={
-        "all": 60_317_116,
-    },
-    fn_load=ourworldindata.create_loader_function("IT"),
+    region_name=IT_REGION_NAMES,
+    region_population=IT_REGION_POPULATION,
```

Comment:
Just like for `region_name` you can optionally pass another dict with short-names:
https://github.com/rtcovidlive/rtlive-global/blob/master/rtlive/sources/data_fr.py#L273

You may need to `git fetch` `git merge master`, because this option was just added on Saturday.

Then we'll use these names in plots/visualizations.

Reply:
I second this comment: currently, people will see regions' codes in the main plot (1, 2, 3...), which I guess are not commonly used in day-to-day Italian life? A short code as we did for France would probably make the graph more explicit at first glance

Reply:
I realize now (!) that I do not think we have standard abbreviations for regions in Italy. Looking around I found [this](https://www.circolocalabrese.org/geography/standard-abbreviations-of-italian-regions-and-provinces.asp), which does not look that official and/or reliable, but it could work (it is just the first 3 letters of region name); for the two autonomous provinces appearing in the data we do have a standard code (TN, BZ).

Reply:
Yeah we did kind of the same for France: three-letter abbreviations

Reply:
I've do as you have suggested in the last commit